app/src/main/java/com/asfoundation/wallet/home/ui/list/header/notifications/AppcoinsAppListModel.kt

```diff
@@ -1,9 +1,8 @@
-package com.asfoundation.wallet.home.transactions.models.appcoinsapps
+package com.asfoundation.wallet.home.ui.list.header.notifications
 
 import com.airbnb.epoxy.EpoxyModel
 import com.airbnb.epoxy.EpoxyModelGroup
 import com.asf.wallet.R
-import com.asfoundation.wallet.home.transactions.models.DefaultCarouselModel_
 import com.asfoundation.wallet.ui.appcoins.applications.AppcoinsApplication
 import com.asfoundation.wallet.ui.widget.holder.ApplicationClickAction
 
```

Comment:
`AppcoinsAppListModel` is unused, maybe remove it? and in that case also `AppCoinsAppModel`? or do you think they will be used again in the future?